grr/server/grr_response_server/bin/grrafana.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+# Lint as: python3
+from absl import app, flags
+import json
+import os
+from typing import Any, cast, Dict, List, Text, Tuple
+
+from fleetspeak.src.server.proto.fleetspeak_server import resource_pb2
+
+from grr_response_core import config
+from grr_response_core.config import server as config_server
+from grr_response_core.config import contexts
+from grr_response_server import fleetspeak_connector
+from grr_response_server import fleetspeak_utils
+from grr_response_server import server_startup
+
+from werkzeug import exceptions as werkzeug_exceptions
+from werkzeug import routing as werkzeug_routing
+from werkzeug import serving as werkzeug_serving
+from werkzeug import wrappers as werkzeug_wrappers
+from werkzeug.wrappers import json as werkzeug_wrappers_json
+
+
+AVAILABLE_METRICS = [
+    'mean_user_cpu_rate',
+    'max_user_cpu_rate',
+    'mean_system_cpu_rate',
+    'max_system_cpu_rate',
+    'mean_resident_memory_mib',
+    'max_resident_memory_mib',
```

Comment:
Almost all strings in this file use double quotes (as is the rest of the GRR codebase) and I don't think there is a compelling reason to do it differently here.